kore/test/Test/Kore/Step/Function/Integration.hs

```diff
@@ -1028,5 +1032,42 @@ evaluate
     -> IO (Pattern Variable)
 evaluate functionIdToEvaluator patt =
     SMT.runSMT SMT.defaultConfig emptyLogger
-    $ evalSimplifier Mock.env { simplifierAxioms = functionIdToEvaluator }
+    $ evalSimplifier env { simplifierAxioms = functionIdToEvaluator }
     $ TermLike.simplify patt
+
+env :: Env
+env =
+    Env
+        { metadataTools       = tools
+        , simplifierTermLike  = Mock.termLikeSimplifier
+        , simplifierPredicate = Mock.predicateSimplifier
+        , simplifierAxioms    = Mock.axiomSimplifiers
+        }
+  where
+    tools =
+        Mock.makeMetadataTools
```

Comment:
IIRC, `Mock.makeMetadataTools` does some things that are sketchy, and I intentionally avoided it for this module. Why is this necessary?